build.assets/tooling/lib/eventschema/ignored.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2023 Gravitational, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package eventschema
+
+type stringSet map[string]struct{}
+
+var ignoredFields = map[string]stringSet{}
```

Comment:
If we don't have use-cases for other markers, I'll just go with a regexp. If we have multiple markers, it might be better to use a library like the controller utils.

Is that an issue if I do this in another PR to make the changes reviewable?